Description: Fix FTBFS with gcc-15
 xosd.c: In function 'display_info':
 xosd.c:973:29: error: passing argument 3 of 'pthread_create' from incompatible pointer type [-Wincompatible-pointer-types]
   973 |   pthread_create(&id, NULL, display_info_driver, &i);
       |                             ^~~~~~~~~~~~~~~~~~~
       |                             |
       |                             void * (*)(void)
 In file included from intern.h:22,
                  from xosd.c:20:
 /usr/include/pthread.h:204:36: note: expected 'void * (*)(void *)' but argument is of type 'void * (*)(void)'
   204 |                            void *(*__start_routine) (void *),
       |                            ~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~
 xosd.c:911:1: note: 'display_info_driver' declared here
   911 | display_info_driver()
       | ^~~~~~~~~~~~~~~~~~~
Forwarded: no
Author: Vincent Cheng <vcheng@debian.org>
Last-Update: 2025-09-23

--- a/src/libxosd/xosd.c
+++ b/src/libxosd/xosd.c
@@ -908,7 +908,7 @@
 
 /* display_info_driver -- main for pthreads in display_info */
 void* 
-display_info_driver() 
+display_info_driver(void* unused)
 {
   xosd * screen = xosd_create(1);
   int nscreens = screen->nscreens;
